quantum/quantum.c

```diff
@@ -58,12 +58,17 @@ float bell_song[][2] = SONG(TERMINAL_SOUND);
 #    include "process_auto_shift.h"
 #endif
 
-static void do_code16(uint16_t code, void (*f)(uint8_t)) {
+#ifdef KEY_OVERRIDE_ENABLE
+extern bool process_key_override(const uint16_t keycode, const keyrecord_t *const record);
+extern void matrix_scan_key_override(void);
```

Comment:
These should be in a header file for the feature (`key_override.h`), and not prototyped here.

Reply:
The reason I didn't expose these functions in `key_override.h` is because they are meant to be used only internally in quantum and should not be exposed in keymaps and userspace (to avoid people calling these functions because they pop up in autocompletion). I'll move them to a new header `key_override_private.h`.

Reply:
A good number of other similar features are in the same boat, but still expose the stuff. 

But this should be okay, I think.